README.md

```diff
@@ -120,8 +120,8 @@ git submodule update --init --recursive
 OpenConsole.sln may be built from within Visual Studio or from the command-line using MSBuild. To build from the command line:
 
 ```shell
-nuget restore OpenConsole.sln
-msbuild OpenConsole.sln
+.\tools\razzle.cmd
+msbuild

```

Comment:
_mega nit_: could we change L124 to `bcz` while we're here?

Reply:
done, and I had to add a change to bcz so it could handle spaces in MSBUILD